daceml/onnx/op_implementations/fpga_implementations.py

```diff
@@ -886,6 +886,1034 @@ def make_compute(sdfg, state, vec_width=1):
         return new_sdfg
 
 
+@op_implementation(op="Conv", name="fpga_tiled")
```

Comment:
Maybe we should  have only one implementation Instead of two (this and the non-tiled).

Any opinion on this?
I see that this one supports only squared kernels... but I don't know how common is to have non-squared ones

Reply:
Personally, for 2d image data I have never seen non-square kernels. The non-tiled version however still has an edge over this implementation on Intel FPGAs. No buffering I/O overhead, because we can use unrolled I/O (However, I think as we discussed the buffering overhead could be hidden in the future with a double-buffering scheme). Vectorization width i.e. width of the unrolling can be chosen more flexibly as by using unrolled I/O with dynamic accesses it is independent of the data dimensions, the aligned vectors depend on the dimensions of the data to be divisible.

Reply:
Theoretically I also have a tiled version with unrolled I/O, which could replace the untiled unrolled I/O version. Would be another 1000 line PR, which we could do later.

Reply:
Ok, leave it as it is for the moment, then we will discuss this later